libs/utils/src/ballot_styles.ts

```diff
@@ -36,6 +37,7 @@ export function generateBallotStyleId(params: {
   languages: LanguageCode[];
   party?: Party;
 }): BallotStyleId {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
did you need to add this for it to clear lint? You didn't change the next line so checking. I was having issues with this lint rule on my PR (I dont remember if it was here specifically)

Reply:
Yeah I couldn't pass lint without this comment